src/MainWindow.cpp

```diff
@@ -736,6 +736,23 @@ MainWindow::OnClose() noexcept
 void
 MainWindow::OnPaint(Canvas &canvas) noexcept
 {
+  
+  // const Look &look = UIGlobals::GetLook();
+
+  // bool inversed = look->info_box.inverse;
+  bool inversed{false};
+  Color color_separator_top;
```

Comment:
If you move this variable into the scope where it is actually used, you can avoid the `look != nullptr` check and reduce this whole block of code to just 1 line.

Reply:
I didn't have time to see this part... but I don't understand clearly understand which scope.. 
I will check this in the next days.

to speed up my understanding, I am interested in additional clues